src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -44,18 +45,20 @@
   private static final String APPLICATION_NAME = "SAY";
```

Comment:
wrong order for the variable definition?

Reply:
I wouldn't worry too much about ensuring that variables are alphabetically ordered.

In alot of cases it's more useful to make variables grouped by their intended purpose rather than their alphabetic order.